SQL1/Homework1-SQL.sql

```diff
@@ -0,0 +1,173 @@
+CREATE DATABASE ZooManagement5;
+
+USE ZooManagement5;
+
+CREATE TABLE Animals1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfAnimal VARCHAR(30),
+    NumberOfAnimal INT,
+);
+
+CREATE TABLE Species1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    NameOfSpecies VARCHAR(20),
+    NumberOfSpecies INT,
+    AnimalsId INT REFERENCES Animals1(Id),
+);
+
+CREATE TABLE CareTakers1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    FirstName VARCHAR(30),
+    LastName VARCHAR(30),
+    AreaOfWork VARCHAR(20),
+    AnimalsId INT REFERENCES Animals1(Id),
+);
+
+CREATE TABLE ZooAreas1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    NameOfTheArea VARCHAR(20),
+    CareTakersId INT REFERENCES CareTakers1(Id),
+);
+
+CREATE TABLE Visitors1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    DayOfTheWeek VARCHAR(30),

```

Comment:
To make sure that only valid information is inputted for the DayOfTheWeek then a separate tables called WeedDays (storing "Monday", "Tuesday", etc.) can be used and the Id of these days can be referenced in this one.